code/modules/projectiles/projectile/magic.dm

```diff
@@ -374,3 +374,12 @@
 	flag = "magic"
 	dismemberment = 50
 	nodamage = 0
+
+/obj/item/projectile/magic/arcane_barrage
+	name = "arcane bolt"
+	icon_state = "arcane_barrage"
+	damage = 20
+	damage_type = BURN
+	nodamage = 0
+	armour_penetration = 100
```

Comment:
Does it really need armor pen at all? 20 burn is very good on its own, and armor types don't have that high of a resistance to it iirc